apps/emqx_exhook/src/emqx_exhook_mgr.erl

```diff
@@ -329,10 +330,14 @@ handle_cast(_Msg, State) ->
 handle_info({timeout, _Ref, {reload, Name}}, State) ->
     {_, NState} = do_reload_server(Name, State),
     {noreply, NState};
-handle_info(refresh_tick, State) ->
+handle_info(refresh_tick, #{servers := Servers} = State) ->
     refresh_tick(),
     emqx_exhook_metrics:update(?REFRESH_INTERVAL),
-    {noreply, State};
+    NServers = maps:map(
```

Comment:
Note: For concurrent health-checking with a long list of servers, emqx_utils:pmap/2,3 would be a better choice.